### PR TITLE
Update changelog for 0.26.3-insiders4:

### PR DESCRIPTION
**Pull request by Colengms**

### CODE REVIEW

**Review by michelleangela — APPROVED** [review]

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.26.3-insiders4: January 17, 2020
+### Bug Fixes
+* Update dependent modules to address a security warning due to use of a deprecated API.  [#4381](https://github.com/microsoft/vscode-cpptools/issues/4381)
+* Fix an issue in which files may be unnecessarily removed from the tag parser database on startup, if using a custom configuration provider, resulting in a large number of files being reparsed.  [@4802](https://github.com/microsoft/vscode-cpptools/issues/4802)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: #4802

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+* Fix an issue in which `Build and Debug Active File` would fail to detect a compiler, despite a compiler present in `compilerPath`.  [#4834](https://github.com/microsoft/vscode-cpptools/issues/4834)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: despite should be "without"

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (Colengms) [reply Extension/CHANGELOG.md]: Should I add use of the clang-format on the system, or wait until after insiders4?

**Review by sean-mcmanus — APPROVED** [review]

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
The date should be the 16th.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**